README.md

```diff
@@ -0,0 +1,13 @@
+# FlowForge
+
+FlowForge is an open-source low-code platform based on the OpenJS Foundation Node-RED
+project. It provides the tools you need to run and manage multiple Node-RED instances
+in your own environment.
+
+## Links
+
+- [Documentation](https://github.com/flowforge/flowforge/blob/add-readme-back/docs/README.md)
+- [Contributing](https://github.com/flowforge/flowforge/blob/add-readme-back/CONTRIBUTING.md)
+- [Code of Conduct](https://github.com/flowforge/flowforge/blob/add-readme-back/CODE_OF_CONDUCT.md)
+- [Security](https://github.com/flowforge/flowforge/blob/add-readme-back/SECURITY.md)
+- [License](https://github.com/flowforge/flowforge/blob/add-readme-back/LICENSE)
```

Comment:
```suggestion
- [Documentation](docs/README.md)
- [Contributing](CONTRIBUTING.md)
- [Code of Conduct](CODE_OF_CONDUCT.md)
- [Security](SECURITY.md)
- [License](LICENSE)
```

Reply:
Ah - just saw your previous comment about why they have to be full URLs.

In which case, they need to link to `main` not `add-readme-back`.